setup.py

```diff
@@ -43,12 +46,16 @@
 """
 
 # defaults
-ISRELEASED = False
+ISRELEASED = True
```

Comment:
Should `ISRELEASED = True` go to master, or is it for release only?  Not urgent---if it ends up on master, we'll fix it after release.

Reply:
I think it should be on a release only. I don't know the history of isreleased, might still be from @avventi, and haven't seen him around for years.